Caiti branch ii

laikasghost opened the pull request:
I totally forgot to create this pull request over the weekend, sorry!

SiennaC committed:
Reverted inventory item art to the orginal version (no excess whitespace).

Commit:
Changed texture scaling

changed from square to proportional

Commit:
Merge branch 'CaitiBranchII' of https://github.com/GlowLimeGames/Deathless into CaitiBranchII

SiennaC reviewed (changes requested):
This still has the squashed images on Windows, but it works if I change the CursorMode to CursorMode.ForceSoftware in Util.SetCursor.

Could you please make this change and then test to see if it also works on Mac?

Commit:
Merge branch 'master' into CaitiBranchII

Commit:
Cursor bug bug fix

Changed CursorMode.Auto to CursorMode.ForceSoftware so it will work cross-platform
